launcher/BaseInstance.cpp

```diff
@@ -252,7 +253,7 @@ int64_t BaseInstance::lastTimePlayed() const
         QDateTime timeNow = QDateTime::currentDateTime();
         return m_timeStarted.secsTo(timeNow);
     }
-    return settings()->get("lastTimePlayed").toLongLong();
+    return m_settings->get("lastTimePlayed").toLongLong();
 }
```

Comment:
Total time played and last time played no longer show up in the bottom of the window when an instance is selected, unless you open the instance's settings, select another instance, and then switch back. The settings might need to be loaded when the instance is selected.

Reply:
oops, looks like the problem was caused by the overrides being in the type-specific section while the normal settings were on the BaseInstance. Changing the override location seemed to fix it for me!
thanks! :)